src/components/Accordion.tsx

```diff
@@ -83,6 +83,7 @@ const StyledEmoji = styled(Emoji)`
 `
 
 export interface IProps {
+  children?: React.ReactNode
```

Comment:
`children` was removed from `React.FC`. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210

I'm starting to think about not using `React.FC` anymore since it doesn't bring any major benefit tbh. (CRA already removed it form their templates https://github.com/facebook/create-react-app/pull/8177)

Reply:
This is interesting, thanks for dropping that link